Resources/Prototypes/_Shitmed/Recipes/Lathes/robotics.yml

```
@@ -0,0 +1,80 @@
+
+- type: latheRecipe
+  id: BasicCyberneticEyes
+  result: BasicCyberneticEyes
+  category: Robotics
+  completetime: 5
+  materials:
+    Steel: 1000
+    Glass: 500
+    Plastic: 500
+    Gold: 300
+    Silver: 300
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)